assertions/src/main/java/io/camunda/zeebe/process/test/inspections/FormInspectionsUtility.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2021 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.camunda.zeebe.process.test.inspections;
+
+import io.camunda.zeebe.process.test.assertions.BpmnAssert;
+import io.camunda.zeebe.process.test.filters.StreamFilter;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.value.deployment.FormMetadataValue;
+import java.util.Comparator;
+import java.util.Optional;
+import java.util.stream.Stream;
+
+public class FormInspectionsUtility {
+
+  /**
+   * Finds if a form is created.
+   *
+   * @param formId the id of the form
+   * @return true if the id of the form is found in the stream, false otherwise
+   */
+  public static boolean isFormCreated(final String formId) {
```

Comment:
🔧 It looks like to me that methods that asserts something in the form of `hasSomething` or `isDone` should be located in the assertion class. utility classes used for retrieving something from the stream. I would be better if we move this method into `FormAssert` class.